src/language-js/parser-babel.js

```diff
@@ -59,12 +59,21 @@ function resolvePluginsConflict(
 }
 
 function createParse(parseMethod, ...pluginCombinations) {
-  return (text, parsers, opts) => {
+  return (text, parsers, opts = {}) => {
+    if (
+      opts.parser === "babel" &&
+      (text.startsWith("// @flow") ||
```

Comment:
> You can also use the form `/* @flow */` for the flag as well.

https://flow.org/en/docs/usage/#toc-prepare-your-code-for-flow

Also we should support `@noflow`: https://flow.org/en/docs/config/options/#toc-max-header-tokens-integer

Also, can't we reuse the logic (`jest-docblock`) used for `--require-pragma` here?

Reply:
All done.